repair/repair.cc

```diff
@@ -2161,7 +2161,10 @@ future<> repair::tablet_repair_task_impl::run() {
                 auto data_centers = std::vector<sstring>();
                 auto hosts = std::vector<sstring>();
                 auto ignore_nodes = std::unordered_set<gms::inet_address>();
-                bool hints_batchlog_flushed = false;
+                auto my_address = erm->get_topology().my_address();
+                auto participants = std::list<gms::inet_address>(m.neighbors.all.begin(), m.neighbors.all.end());
+                participants.push_front(my_address);
+                bool hints_batchlog_flushed = co_await flush_hints(rs, id, rs._db.local(), m.keyspace_name, tables, ignore_nodes, participants);
```

Comment:
Why do we need to flush hints batchlog for repair gc to work?

Reply:
- hints writes to nodes
- repair without flush the hints
- compaction drops tombstone and covered data after repair
- hints are sent
- data resurrected

Reply:
I see, thanks.